x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/space_aware_privilege_section/privilege_space_table.tsx

```diff
@@ -222,6 +224,10 @@ export class PrivilegeSpaceTable extends Component<Props, State> {
         actions: [
           {
             render: (record: TableRow) => {
+              if (privilegeCalculator.isWildcardBasePrivilege(record.privilegeIndex)) {
+                return <></>;
+              }
+
```

Comment:
The changes here seem to overlap with https://github.com/elastic/kibana/pull/181400

Can we isolate this PR to only changes for the new endpoint, and isolate the changes for supporting application wildcard in the UI to 181400?

Reply:
Reverted changes merged from https://github.com/elastic/kibana/pull/181400